category/vm/compiler/ir/x86.cpp

```diff
@@ -27,13 +27,10 @@
 
 #include <asmjit/core/jitruntime.h>
 
-#include <quill/detail/LogMacros.h>
-// clang-tidy incorrectly views this macro as unused
-#include <quill/Quill.h> // IWYU pragma: keep
+#include <quill/Quill.h>
```

Comment:
This clang-tidy fix is now global in the configuration file; drive-by cleanup here